bin/GameIndex.yaml

```diff
@@ -36787,7 +36787,7 @@ SLUS-21381:
   clampModes:
     vuClampMode: 2
 SLUS-21382:
-  name: "Franklin - A Birthday Surprise"
+  name: "Franklin the Turtle - A Birthday Surprise"
```

Comment:
Looks more like "Franklin-Turtle: Franklin - A Birthday Surprise" to me, or just how it was

https://my-test-11.slatic.net/p/fffb15b474ff4bed5549806587dba0c6.jpg

Reply:
![image](https://user-images.githubusercontent.com/24227051/108631949-c4160700-746c-11eb-98ec-d1fbc21fb8d1.png)

Enhance :)

Reply:
huh, it does say "the", but still, it has "Franklin" again as part of the main title, so it's still wrong ;)